config/settings/base.py

```diff
@@ -419,3 +420,8 @@
 MATRIMONY_SENDER_EMAIL = env("MATRIMONY_SENDER_EMAIL", default="admin@example.net")
 PROFILE_ID_PREFIX = env("PROFILE_ID_PREFIX", default="VMB")
 TABBED_ADMIN_USE_JQUERY_UI = env.bool("TABBED_ADMIN_USE_JQUERY_UI", default=True)
+
+# Django Recaptcha:
+# ------------------------------------------------------------------------------
+RECAPTCHA_PUBLIC_KEY = "6Ld7vz0hAAAAAKPNoffaypjDLK6aHJ-NC8BexW7Y"
```

Comment:
We should read not commit keys in the repo. We should read it from env using `env(..)` calls as you see above